srcpkgs/PolyMC/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'PolyMC'
+pkgname=PolyMC
+version=1.1.0
+revision=1
+create_wrksrc=yes
+_nbtversion="dc72a20b7efd304d12af2025223fad07b4b78464"
+_quazipversion="09ec1d10c6d627f895109b21728dda000cbfa7d1"
+build_style=cmake
+configure_args="-DLauncher_BUILD_PLATFORM=Void -DLauncher_PORTABLE=0
+ -DLauncher_VERSION_BUILD=${revision}"
+hostmakedepends="openjdk11 qt5-host-tools qt5-qmake"
+makedepends="qt5-devel"
+depends="virtual?java-runtime"
+short_desc="Custom launcher for Minecraft"
+maintainer="Philipp David <pd@3b.pm>"
+license="GPL-3.0-or-later"
+homepage="https://polymc.org/"
+distfiles="https://github.com/PolyMC/PolyMC/archive/${version}.tar.gz
+ https://github.com/MultiMC/libnbtplusplus/archive/${_nbtversion}.tar.gz
+ https://github.com/stachenov/quazip/archive/${_quazipversion}.tar.gz"
+checksum="4114b11a47a5380c811df890310578ddb3457f0a96caf019f388e1f551d58e99
+ 2ed9dc824007b4f7b12f807f56c98e1248b8a7aba7eca4838085092958dc1595
+ 618700e4e5f1f884e22bc8ed2c2c102305711ebf5a39296bf4c6017a4e6d4e35"
+
+case "$XBPS_TARGET_MACHINE" in
+	armv*) broken="https://github.com/MultiMC/MultiMC5/issues/2895";;
+esac
+
+post_extract() {
+	find PolyMC-${version}/ -mindepth 1 -maxdepth 1 -exec mv {} . \;
+	rmdir PolyMC-${version}
```

Comment:
Use `build_wrksrc` instead please.

Reply:
That also breaks since the patch is copied to and referenced from $wrksrc...

Reply:
I solved this in #35591 by adding build_wrksrc to the path in the patch. not ideal at all but it works, I guess

Reply:
I suppose that will work, since I will try to get the GITDIR-NOTFOUND detection upstreamed and the other change is ideal for vsed.

Reply:
The change has been upstreamed.